### ISSUE TITLE
Can't add jobs on project without repository

### ISSUE THREAD

**Issue by quickslyver** [jbox-web/redmine_jenkins#47]
It is impossible to add jenkins jobs on a redmine project without at least one repository configured.

My current workaround is to add a "dummy" filesystem repository to the project.

**Comment by n-rodriguez** [comment]
> It is impossible to add jenkins jobs on a redmine project without at least one repository configured.

Yep. This is 'a feature' as it permits to link project's issues with the job.
